packages/docusaurus-plugin-content-blog/src/index.js

```diff
@@ -82,19 +80,7 @@ class DocusaurusPluginContentBlog {
         );
 
         const fileString = await fs.readFile(source, 'utf-8');
-        const {metadata: rawMetadata, content} = parse(fileString);
-
-        let truncatedSource;
-        const isTruncated = TRUNCATE_MARKER.test(content);
```

Comment:
Instead of generating new files for truncated version of blogpost, we should just use Webpack resourceQuery (e.g: `import('blog/212.md?truncated=true')`) such that our webpack loaders can understand whether to truncate the content or not.